client/webserver/site/src/js/markets.ts

```diff
@@ -94,6 +94,8 @@ const percentFormatter = new Intl.NumberFormat(document.documentElement.lang, {
 
 const parentIDNone = 0xFFFFFFFF
 
+const colorPink = 'pink'
```

Comment:
I'm not feeling this color.

Reply:
> I'm not feeling this color.

Changed to red, or do you have something else in mind ?

Reply:
I'd start by seeing if any of the constants in **application.scss** would work. Two that jump out at me are

```SCSS
$sellcolor_dark: #e95e5e;
$sellcolor_light: #99302b;
```

And then set the outline color in both **market.scss**.

```SCSS
div[data-handler=markets] {
  input {
    outline: 2px solid $sellcolor_light;
  }
}
```

and **market_dark.scss**.

```SCSS
body.dark {
  div[data-handler=markets] {
    input {
      outline: 2px solid $sellcolor_dark;
    }
  }
}
```

Reply:
Actually, I don't know that would work with the animation. I suppose you would need to define the rgb values in **markets.ts** after all. You can pick which color (dark/light) using `State.isDark()`.

Reply:
Actually, I don't know that would work with the animation. I suppose you would need to define the rgb values in **markets.ts** after all. You can pick which color (dark/light) using `State.isDark()`.

Reply:
Yeah, I don't think I can make css can work with animation, or at least don't know how to make it workable and simple,

I went the route you've suggested [below](https://github.com/decred/dcrdex/pull/1921#discussion_r1017804446)